ws-nextjs-app/cypress/e2e/articlePage/testsForCanonicalOnly.ts

```diff
@@ -211,34 +211,60 @@ export default ({
       }
     });
 
-    describe.skip('Continue Reading Button', () => {
-      it('should render the Continue Reading button', () => {
+    describe('Continue Reading Button', () => {
+      it('should render the Continue Reading button', function test() {
+        runIfToggleEnabled({
+          service,
+          toggleName: 'continueReadingButton',
+          testContext: this,
+        });
+
+        cy.reload();
```

Comment:
Force `reload` so that the button appears again. This is to counter-act the `beforeEachFns` that automatically clicks the button before each test.